src/Catalyst.TestUtils/TransactionHelper.cs

```diff
@@ -67,5 +68,49 @@ public static class TransactionHelper
             };
             return transaction;
         }
+        
+        public static TransactionBroadcast GetContractTransaction(ByteString data,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method has 12 parameters, which is greater than the 7 authorized.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4350015)